test/testdata/workspaces/gradle-sample-app/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM adoptopenjdk/openjdk11:alpine-jre
+FROM eclipse-temurin:17-jdk-alpine

```

Comment:
for consistency.... shouldn't we use the red hat image instead?

Reply:
@stitakis we actually need to move all quickstarters to UBI

Reply:
Agree, we should probably use the RH image here as well.

Two notes though:

1. this is a test, not a quickstarter. this repo does not contain quickstarters. If I'm not mistaken this file isn't even used in the test and only there fore completeness
2. I was recently wondering if https://github.com/GoogleContainerTools/distroless wouldn't be a good default choice as well. Especially for statically compiled binaries (hello Go again), RH does not provide an image with such a small attack surface ...

Reply:
Will change this to `registry.access.redhat.com/ubi8/openjdk-17-runtime:1.10`

Reply:
@michaelsauter very interesting these distroless images... are you suggesting to use them only for tests or also as  application runtime?

Reply:
@stitakis Application runtime. I have no real-world experience with them though, but if I were to deploy a Go application right now, I would consider basing it on one of those images. In theory this should prevent many redeploys due to vulnerabilities, which is especially helpful in a regulated context where any redeployment is "costly". But this might be a conversation for another GH issue ;)

Reply:
@michaelsauter I apologise, I was commenting on the wrong repo!

Reply:
@metmajer nono, this is the right repo :D we'd love to have your comments here!